src/iface/fragments.rs

```diff
@@ -0,0 +1,170 @@
+use storage::Assembler;
+use time::Instant;
+use managed::ManagedSlice;
+
+/// Default timeout duration
+const FRAGMENTATION_TIMEOUT_MS: i64 = 500;
+
+#[allow(dead_code)]
```

Comment:
Better not include the module at all if built without the fragmentation feature.

Reply:
But I need to include `FragmentSet` because interface now contains `fragments:              Option<FragmentSet<'e>>`, except without enabled fragmentation it compiles away as it is always None